### DISCUSSION TITLE
Does the custom search function support "or" ？

### DISCUSSION THREAD

**Discussion by toyo2333** [FreshRSS/FreshRSS#5627] [Q&A] [answered]
For example, how do I search for content that contains "USA" or "Washington"?

Another question, how do I set up a search that only searches the title of the feed?

**Comment by Alkarex** [comment, marked as answer]
Hello,
Please check our documentation: https://freshrss.github.io/FreshRSS/en/users/10_filter.html#with-the-search-field

1) Try: `USA OR Washington`
2) Try: `intitle:something`

**Reply by toyo2333** [reply]
Thanks！
